adapters/appnexus_test.go

```diff
@@ -276,7 +276,8 @@ func TestAppNexusBasicResponse(t *testing.T) {
 		}
 
 		pbin.AdUnits[i] = pbs.AdUnit{
-			Code: tag.code,
+			Code:       tag.code,
+			MediaTypes: []string{"BANNER"},
```

Comment:
There's no assertion that you received a `banner` type bid response. Should also do the same for video